overrides/partials/header.html

```diff
@@ -0,0 +1,72 @@
+{#-
```

Comment:
@ruthkoole. Als je templates gaat editen is het moeilijker om naar een andere template over te gaan. Iedere template gebruikt namelijk een andere structuur. Dus de eigenaar van de styling moet wel zeker zijn dat ze deze template willen blijven gebruiken als je het gaat editen.

Reply:
Ja, dit niet chill. Maar kon geen geschikte partial vinden om te extenden.

We zouden `partials/search.html` kunnen overwegen.